pkg/printers/internalversion/describe.go

```diff
@@ -1321,6 +1321,16 @@ func describeContainerBasicInfo(container api.Container, status api.ContainerSta
 			w.Write(LEVEL_2, "Port:\t%s\n", portString)
 		}
 	}
+	hostPortString := describeContainerHostPorts(container.Ports)
+	if strings.Contains(hostPortString, ",") {
+		w.Write(LEVEL_2, "HostPorts:\t%s\n", hostPortString)
+	} else {
+		if len(hostPortString) == 0 {
+			w.Write(LEVEL_2, "HostPort:\t<none>\n")
+		} else {
+			w.Write(LEVEL_2, "HostPort:\t%s\n", hostPortString)
```

Comment:
I think that having more `if` conditions just for respecting plural form doesn't worth it. This code also could benefit from using `stringOrNone()`:
```golang
w.Write(LEVEL_2, "HostPorts:\t%s\n", stringOrNone(hostPortString))
```

Reply:
@php-coder Excellent.

Updated. PTAL. Thanks.